definitions/events/identity/identity_created.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package identity;
+
+option go_package = "github.com/blue-health/blue-proto/generated/go/events/identity";
+
+message IdentityCreated {
+  string author_id = 1;
+  string source = 2;
```

Comment:
```suggestion
  string author_id = 1;
  string subject_id = 2;
  string source = 3;
```
Let's add the author and subject IDs, in case some admin user creates an account in the future